### PR TITLE
Create guidlines for contributors

### LINKED ISSUE #133: Specify the process to extend the API specification...
... as README.md in the repository, based on https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community .

### PR DESCRIPTION
**Pull request by christoph-maurer**
I have created a first version of CONTRIBUTING.md to describe the procedure for questions, bugs and enhancements.

I have extended README.md to make it easier for contributors to get to know the specification.

Closes #133

### CODE REVIEW

**Comment by christoph-maurer** [comment]
@simon-wacker Do you think we should already use Github Discussions?

**Comment by simon-wacker** [comment]
I have no experience with GitHub Discussions. It sounds like a good idea.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+When you receive positive feedback, please fork the repository and show us how you would remove the bug. Please [test your solution](#test-your-contribution) and create a [GitHub Pull Request](https://github.com/ise621/building-envelope-data/compare) to propose your solution and describe the problem and the solution clearly. Please refer to the issue which is solved by your solution.
+
+In this way, you contribute to a fast release of extended versions.
+
+## Test your contribution
+
+Please verify that your commits pass the following tests:
+
+- [ ] Run `make shell` and `make format` to format branch with [Prettier](https://prettier.io).
```
Comment (simon-wacker) [inline CONTRIBUTING.md:41]: You run `make shell` to drop into a shell within a fresh Docker container (this is only needed if you use Docker). You can then run `make format` to format the code. Formatting code is not a test --- it polishes the code and makes it adhere to our coding styles.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+- [ ] Use `make compile` to verify that all schemas are valid against the specification of GraphQL and JSON Schema.
+- [ ] Use `make example` and verify that all examples are valid against the schemas.
```
Comment (simon-wacker) [inline CONTRIBUTING.md:43]: Wording should `Run` instead of `Use` and `to verify` instead of `and verify`. Same below and above.

File: README.md [hunk 3]
```diff
@@ -15,7 +36,17 @@ An optical data set can include metadata like the [standard](https://github.com/
...
+
+### For beginners
+
+With you web browser, you can search our [wiki](https://github.com/ise621/building-envelope-data/wiki), the [issues](https://github.com/ise621/building-envelope-data/issues) and [pull requests](https://github.com/ise621/building-envelope-data/pulls) and contribute to them.
+
+In order to browse the code conveniently, you should first [clone the repository](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/cloning-a-repository) and then use a text editor, for example [Visual Studio Code](https://code.visualstudio.com/).
+
+### On your Linux machine
+
+In order to use all functions of this code for example to format and test, please follow the instructions below.
```
Comment (simon-wacker) [inline README.md:49]: Rephrase to: `In order to use our development tooling, for example, to format code and to run tests, follow the instructions below.`

**Review by simon-wacker — CHANGES_REQUESTED** [review]

**Comment by christoph-maurer** [comment]
@simon-wacker I addressed the three comments of your review with eb8af6bae3b3600b91442ebc8133ca46cd33dbd7.

**Review by simon-wacker — APPROVED** [review]

**Pull request merged**